index.html

```diff
@@ -66,7 +66,258 @@ <h1 class="header-title ">I'm Abigail T Glad to see you!
+                <p class="card-info-item">2015</p>
+              </div>
+
+              <p class="card-description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+
+              <ul class="card-list">
+                <li class="card-list-tags-html">html</li>
+                <li class="card-list-tags-css">css</li>
+                <li class="card-list-tags-js">Javascript</li>
+              </ul>
+
+              <a class="card-link" href="#">See Project</a>
+              
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+</section>
+
+<!-- top section -->
+<section class="top-section">
+      
+  <h1 class="top-title "> About Myself</h1>
+
+  <p class="header-paragraph ">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+  
+  <a class="header-ref" href="">LET'S CONNECT</a>
+
+  <div class="class-outer">
+
+      <ul class="list-wrapper">
+        <li>
+          <a href="#">
+            <img src="./images/sm-icons/github.png" alt="Github">
+          </a>
+        </li>
+
+        <li>
+          <a href="#">
+            <img src="./images/sm-icons/linkedIn.png" alt="Github">
+          </a>
+        </li>
+
+        <li>
+          <a href="#">
+            <img src="./images/sm-icons/anglist.png" alt="Github">
+          </a>
+        </li>
+
+        <li>
+          <a href="#">
+            <img src="./images/sm-icons/twitter.png" alt="Github">
+          </a>
+        </li>
+
+        <li>
+          <a href="#">
+            <img src="./images/sm-icons/medium.png" alt="Github">
+          </a>
+        </li>
+      </ul>   
+  </div>
+
+  <div class="btn-container">
+    <a href="#" class="ctb-btn">Get my resume</a>
+  </div>
+
+  <!-- Languages -->
+  <div class="languages">
+    <h2 class="lang-title">Language</h2>
+  </div>
+
+  <div>
+    <ul class="about-me-list">
+      <li class="about-me-list-item">
+        <img class="js-image" src="./images/javascript.png" alt="" >
+        <h3>Javascript</h3>
+      </li>
+
+      <li class="about-me-list-item">
+        <img class="js-image" src="./images/html.png" alt="" >
+        <h3>HTML</h3>
+      </li>
+
+      <li class="about-me-list-item">
+        <img class="js-image" src="./images/css.png" alt="" >
+        <h3>CSS</h3>
+      </li>
+    </ul>
+  </div>
+
+  <!-- Framework -->
+  <div class="techies">
+    <h2 class="techies-title">Frameworks</h2>
+    <img class="forward-arrow" src="./images/arrow.png" alt="Arrow Forward">
+  </div>
+  <hr class="hr">
+
+  <!-- Skills -->
+  <div class="techies">
+    <h2 class="skills-title">Skills</h2>
+    <img class="techies-forward-arrow" src="./images/arrow.png" alt="Arrow Forward">
+  </div>
```

Comment:
- [ ] Kindly consider changing your list of technologies to a nested list as demanded by the project requirements. A nested list looks like this;
```html
<ul>
  <li>
    <h2>Languages</h2>
    <ul>
      <li>CSS</li>
      ...
    </ul>
  </li>
  <li>
    <h2>Frameworks</h2>
  </li>

  <li>
    <h2>Skills</h2>
  </li>
</ul>
```

Reply:
Issue resolved. Thanks for the feedback @Kamba56